### ISSUE TITLE
Documentation about Extending a Model Controller has a not defined variable: model

### ISSUE THREAD

**Issue by abdonrd** [strapi/strapi#4504]
**Describe the bug**

The documentation about [Extending a Model Controller](https://strapi.io/documentation/3.0.0-beta.x/concepts/controllers.html#core-controllers) has a not defined variable: `model`.

Example with `findOne`:

```javascript
const { sanitizeEntity } = require('strapi-utils');

module.exports = {
  /**
   * Retrieve a record.
   *
   * @return {Object}
   */

  async findOne(ctx) {
    const entity = await strapi.services.restaurant.findOne(ctx.params);
    return sanitizeEntity(entity, { model });
  },
};
```

What should be the `model`? `strapi.models.restaurant`?

**System**
- Node.js version: `v10`
- NPM version: `v6`
- Strapi version: `v3.0.0-beta.17.5`
- Database: MongoDB
- Operating system: macOS

**Comment by n8tb1t** [comment]
`const model = strapi.models.restaurant`

**Comment by abdonrd** [comment]
Closed by #4530 and #4545.
